Tasks/AzureRmWebAppDeployment/kuduutility.ts

```diff
@@ -133,4 +133,45 @@ export async  function containsParamFile(webAppPackage: string ) {
     }
     tl.debug(tl.loc("Isparameterfilepresentinwebpackage0", isParamFilePresent));
     return isParamFilePresent;
+}
+
+export async function createKuduPhysicalPath(publishingProfile, physicalPath: string) {
+    var defer = Q.defer<string>();
+    physicalPath = physicalPath.replace(/[\\]/g, "/");
+    var kuduPhysicalpathUrl = "https://" + publishingProfile.publishUrl + "/api/vfs/" + physicalPath + "/";
+    var basicAuthToken = 'Basic ' + new Buffer(publishingProfile.userName + ':' + publishingProfile.userPWD).toString('base64');
+    var headers = {
+        'Authorization': basicAuthToken,
+        'If-Match': "*"
+    };
+    tl.debug(tl.loc("RequestedURLforkuduphysicalpath", kuduPhysicalpathUrl))
+
+    httpObj.send('GET', kuduPhysicalpathUrl, null, headers, (error, response, body) => {
+        if (error) {
+            defer.reject(error);
+        }
+        else if (response.statusCode === 200 || response.statusCode === 201 || response.statusCode === 204) {
+            tl.debug(tl.loc("Physicalpathalreadyexists", physicalPath));
+            defer.resolve(tl.loc('Physicalpathalreadyexists'));
```

Comment:
Is this message customer facing? If not remove localization.  I feel this message not required to shown to customer